### PR TITLE
Added PHP 8.0 Support

### LINKED ISSUE #4: PHP 8.0 support
### Feature Request

|    Q        |   A
|------------ | ------
| New Feature | yes

#### Summary

To be prepared for the december release of PHP 8.0, this repository has some additional TODOs to be tested against the new major version.

In order to make this repository compatible, one has to follow these steps:

- [ ] Modify `composer.json` to provide support for PHP 8.0 by adding the constraint `~8.0.0`
- [ ] Modify `composer.json` to drop support for PHP less than 7.3
- [ ] Modify `composer.json` to implement phpunit 9.3 which supports PHP 7.3+
- [ ] Modify `.travis.yml` to ignore platform requirements when installing composer dependencies (simply add `--ignore-platform-reqs` to `COMPOSER_ARGS` env variable)
- [ ] Modify `.travis.yml` to add PHP 8.0 to the matrix (*NOTE*: Do not allow failures as PHP 8.0 has a feature freeze since 2020-08-04!)
- [ ] Modify source code in case there are incompatibilities with PHP 8.0

### PR DESCRIPTION
**Pull request by delassiter**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | yes

### Description

Close #4 - Added PHP 8.0 Support

### CODE REVIEW

**Comment by delassiter** [comment]
Codesniffer fails on License line on every file, and has done so forever.

File: .travis.yml [hunk 1]
```diff
@@ -44,7 +45,7 @@ install:
   - stty cols 120 && composer show
 
 script:
-  - if [[ $TEST_COVERAGE == 'true' ]]; then composer test-coverage ; else composer test ; fi
+  - if [[ $TEST_COVERAGE == 'true' ]]; then XDEBUG_MODE=coverage composer test-coverage ; else composer test ; fi
```
Comment (boesing) [inline .travis.yml]: Does this give any benefit?
There are 0 laminas components which added this and thus I think we can omit it here.

**Review by boesing — APPROVED** [review]
LGTM

I've one question stated in the `.travis.yml`. After this, we can create a release.

File: .travis.yml (hunk 1 above)
Comment (delassiter) [reply .travis.yml]: I'm not to sure where that came from, but it works without it, so I removed it.

**Comment by boesing** [comment]
Thanks, @delassiter!

**Pull request merged**